src/core/hle/service/am/applets/software_keyboard.cpp

```diff
@@ -13,11 +13,23 @@
 
 namespace Service::AM::Applets {
 
+namespace {
+enum class Request : u32_le {
```

Comment:
```suggestion
enum class Request : u32 {
```

This will break on big endian otherwise, given `u32_le` becomes a struct in that scenario.